app/assets/javascripts/components/story/EstimateBadge/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import StoryIcon from '../StoryIcon';
+import StoryEstimate from './StoryEstimate';
+import StoryDescriptionIcon from '../StoryDescriptionIcon';
+import { storyPropTypesShape } from '../../../models/beta/story';
+
+const EstimateBadge = ({
+  story
```

Comment:
This component `props` and `propTypes` does not match. 😢

Reply:
I inserted the wrong name of variable. Thanks!